server/repair_schedule.jl

```diff
@@ -0,0 +1,66 @@
+import Base.in
+
+BestResult = @NamedTuple{shifts::Shifts, score::Number}
+
+function repair_schedule(schedule_data)
+    ITERATION_NUMBER = 20
```

Comment:
te stałe trzeba wyrzucić do innego pliku, nie mogą być zdefiniowane w funkcji, to na pewno